kyuubi-server/src/main/scala/org/apache/kyuubi/events/ServerEventHandlerRegister.scala

```diff
@@ -36,6 +38,17 @@ object ServerEventHandlerRegister extends EventHandlerRegister {
       kyuubiConf)
   }
 
+  override def createKafkaEventHandler(kyuubiConf: KyuubiConf): EventHandler[KyuubiEvent] = {
+    val topic: String = kyuubiConf.get(SERVER_EVENT_KAFKA_TOPIC).getOrElse({
+      throw new IllegalArgumentException(s"${SERVER_EVENT_KAFKA_TOPIC.key} must be configured")
+    })
```

Comment:
Eliminate unnecessary `()`
```suggestion
    val topic: String = kyuubiConf.get(SERVER_EVENT_KAFKA_TOPIC).getOrElse {
      throw new IllegalArgumentException(s"${SERVER_EVENT_KAFKA_TOPIC.key} must be configured")
    }
```